dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/program/EnrollmentServiceTest.java

```diff
@@ -242,11 +244,11 @@ void testGetEnrollmentsByProgram() {
     manager.save(enrollmentA);
     manager.save(enrollmentB);
     manager.save(enrollmentD);
-    List<Enrollment> enrollments = apiEnrollmentService.getEnrollments(programA);
+    List<Enrollment> enrollments = eventProgramEnrollmentService.getEnrollments(programA);
```

Comment:
As the previous PR, I think it make sense to create `EventProgramEnrollmentServiceTest` to test this method.
This test class should be removed when this Jira task is done, but we will need more time to solve the program/tracker event separation.